pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+The aim of this library is to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
+```go
+wi.SetClient(
+    &http.Client{
+        Timeout: time.Second * 2,
+    },
+    nil,
+)
+```
+
+If you service will use any internal function of william like a service, Ej: adding permission direct from you service and not from the Hub you need to set a KeyPair to auth.
+
+```go
+wi.SetKey(
+    "ServiceKeyID",
+    "ServiceKeySecret",
+)
+```
+
+By default William is created with a bypass, you can enable with:
+```go
+wi.Enable()
```

Comment:
WDYT about doing the other way around here? Will.IAM is created normally, but provides the method `Bypass()` to enable testing mode. IMO this approach has a few advantages:

* make explicit that Will.IAM is in Bypass mode in tests
* clients won't need to enable the lib explicitly, as it is already enabled by default

Reply:
Yes, I also think that is the better way.